pyuvdata/uvdata/uvdata.py

```diff
@@ -6477,6 +6490,10 @@ def read_mwa_corr_fits(
             Only used if flag_init is True. Set to True to flag the center fine
             channel of each coarse channel. Only used if file_type is
             'mwa_corr_fits'.
+        remove_flagged_ants : bool
+            Option to perform a select to remove antennas flagged in the metafits
+            file. If flag_small_sig_ants is True then antennas flagged by the Van
```

Comment:
same comment here as for the method in the method in mwa_corr_fits.py

Reply:
fixed